plugins/foreman_statistics/1.0/index.md

```diff
@@ -6,17 +6,18 @@ version: 1.0
 #robots: noindex
 ---
 
-<div class="bs-callout-warning bs-callout">
-  <h4>This is a Work in progress</h4>
-  Plugin will be available only in the next major release!
-</div>
-
 # 1. {{ page.title }}
 
 This plugin enables Foreman to show a nice statistics dashboard and create Trend charts.
 
+## 1.1 Release notes
+
+### Drop the cron job
+
+Foreman statistics 1.0 onwards automates data collection by facilitating a dynflow processing engine and no manual steps are required.
```

Comment:
I'm wondering if we need to mention the version. Versions prior to 1.0 weren't really shipped to users.
```suggestion
Foreman statistics moves trend calculation from a cronjob to a Dynflow job and no manual steps are required.
```